Resources/Prototypes/DeltaV/Entities/Mobs/Player/human.yml

```diff
@@ -15,3 +15,22 @@
     - type: AutoImplant
       implants:
       - DeathAcidifierImplant
+
+- type: entity
+  noSpawn: true
+  parent: MobHuman
+  id: MobHumanSyndicateParamedic
+  name: Syndicate Paramedic
+  suffix: Human
+  components:
+    - type: Loadout
+      prototypes: [SyndicateParamedicGear]
+    - type: RandomMetadata
+      nameSegments: [names_death_commando]
+    - type: NpcFactionMember
```

Comment:
can probably parent syndicate agent to remove the randommeta -> autotraitor dupes (assuming reinforcement has that it probably does)

Reply:
The Syndicate Agent doesn't have auto traitor, the monkey does (which i'll PR to fix on wizden in like 30 minutes)

Reply:
then just parent and add autotratiro the rest is dupe